Makefile

```diff
@@ -1,18 +1,56 @@
 GOTOOLS = \
-					github.com/mitchellh/gox \
-					github.com/Masterminds/glide \
-					gopkg.in/alecthomas/gometalinter.v2 \
-					github.com/gogo/protobuf/protoc-gen-gogo \
-					github.com/gogo/protobuf/gogoproto
-
+	github.com/mitchellh/gox \
+	github.com/Masterminds/glide \
+	gopkg.in/alecthomas/gometalinter.v2 \
+	github.com/gogo/protobuf/protoc-gen-gogo \
+	github.com/gogo/protobuf/gogoproto
+GOTOOLS_CHECK = gox glide gometalinter.v2 protoc protoc-gen-gogo
+PACKAGES=$(shell go list ./... | grep -v '/vendor/')
 INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
 
-all: protoc install test metalinter
+all: check build test install metalinter
 
-PACKAGES=$(shell go list ./... | grep -v '/vendor/')
+check: check_tools get_vendor_deps
```

Comment:
should get_vendor_deps be part of this ? If `check_tools` says `glide` is not installed, `get_vendor_deps` will fail ...

Reply:
if glide is not installed, check_tools will cause Makefile to exit.